automation/pr-to-meta-resin.sh

```diff
@@ -48,12 +48,13 @@ docker-resin-supervisor-disk: Update to ${TAG}
 
 Changelog-Entry: Update supervisor to ${TAG}
 Change-Type: patch
-" <<EOF
+"
+
+	git push origin $HEAD <<EOF
```

Comment:
I think you could also use
```
git push https://${GITHUB_USERNAME}:${GITHUB_PASSWORD}@github.com/resin-os/meta-resin.git $HEAD
```
or you could use an oauth token from https://github.com/settings/tokens as
```
git push https://$OAUTH_TOKEN:x-oauth-basic@github.com/name/repo.git $HEAD
```
which lets you limit the permissions a bit

Alternatively if you use one of those urls in the git clone I don't think you'll have to worry about providing the credentials in the other places of this script